lib/api-client.js

```diff
@@ -6,22 +6,21 @@ var apiClient = new JSONAPIClient(config.host + '/api', {
   'Accept': 'application/vnd.api+json; version=1',
 }, {
   beforeEveryRequest: function() {
+    console.log('using local panoptes-client');
     var auth = require('./auth');
     return auth.checkBearerToken();
   },
 
-  handleError: function(response) {
+  handleError: function(error) {
```

Comment:
if `error` is `undefined` I need to create it as a new Error object so an `error.message` can be added.